### PR TITLE
Add function to create filesystem device

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

### CODE REVIEW

**Comment by yafu-1** [comment]
Test results:
# python
>>> from virttest.utils_test import libvirt_device_utils
>>> fsdev_dict = {}
>>> fsdev = libvirt_device_utils.create_fs_xml(fsdev_dict)
>>> print(fsdev)
<?xml version='1.0' encoding='UTF-8'?>
<filesystem accessmode="passthrough" type="mount"><driver type="virtiofs" /><source dir="/fs_share_dir" /><target dir="fs_mount" /><alias name="ua-b7827dd0-967c-11ea-940d-78e7d1815541" /><binary path="/usr/libexec/virtiofsd" xattr="on"><cache mode="on" /><lock flock="on" posix="on" /></binary></filesystem>

**Comment by yafu-1** [comment]
@dzhengfy @chunfuwen 
Could you help to review this pr please? Thanks.

File: virttest/utils_test/libvirt_device_utils.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import uuid
+
+from virttest.libvirt_xml.devices.filesystem import Filesystem
+
+
+def create_fs_xml(fsdev_dict):
+    """
+    Create filesystem device xml
+
+    :param fsdev_dict: filesystem device paramter dict
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt_device_utils.py:11]: add return key word in doc string

Comment (yafu-1) [reply virttest/utils_test/libvirt_device_utils.py:11]: Updated.

**Pull request merged**